src/Core/Types.Tests/Types/ObjectTypeTests.cs

```diff
@@ -1462,5 +1488,15 @@ public class FooDeprecated
 
             public string Bar2() => "Foo 2: Electric foo-galoo";
         }
+
+        public struct FooStruct
+        {
+            public string GetBar() => "Baz";
```

Comment:
Could you also add both a field and a property to the struct definition? In order to further improve the validity of the new tests :)

Reply:
Sure :)

Reply:
Although we are not inferring fields ... fields are ignored.

Reply:
But I will add a test that ensures that is the case.

Reply:
OK, test are now improved.